commands/build.go

```diff
@@ -643,6 +649,77 @@ func parsePrintFunc(str string) (*build.PrintFunc, error) {
 	return f, nil
 }
 
+func addGitProvenance(labels []string, contextPath string, dockerfilePath string) ([]string, error) {
+	if v, ok := os.LookupEnv("BUILDX_GIT_INFO"); ok && contextPath != "" {
```

Comment:
nit: we should have this be an early exit, so we don't need to indent the rest of this function